Minesweeper.java

```diff
@@ -0,0 +1,99 @@
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionListener;
+import java.util.*;
```

Comment:
do we really want to import everything in `java.util`? i would change this to only import the libraries that are being used